data/json/monsters/kraken_spawn.json

```
@@ -2,7 +2,7 @@
   {
     "id": "mon_octupus_stalker",
     "type": "MONSTER",
-    "name": { "str": "Stalker Octopus", "str_pl": "Stalker Octopi" },
+    "name": { "str": "stalker octopus", "str_pl": "stalker octopi" },
```

```suggestion
    "name": { "str": "stalker octopus", "str_pl": "stalker octopuses" },
```

According to [wiktionary](https://en.wiktionary.org/wiki/octopus)

> The plural [octopi](https://en.wiktionary.org/wiki/octopi#English) is [hypercorrect](https://en.wiktionary.org/wiki/hypercorrect), coming from the mistaken notion that the -us in [octōpūs](https://en.wiktionary.org/wiki/octopus#Latin) is a Latin [second declension](https://en.wiktionary.org/wiki/Appendix:Latin_second_declension) ending.

It seems `octupuses` is both correct and commonly used, while `octupodes` is etymologically correct but rarely used.